.github/workflows/lint.yml

```diff
@@ -85,10 +87,10 @@ jobs:
           php7.4 vendor/bin/parallel-lint --blame --exclude node_modules --exclude vendor .
           php8.0 vendor/bin/parallel-lint --blame --exclude node_modules --exclude vendor .
           php8.1 vendor/bin/parallel-lint --blame --exclude node_modules --exclude vendor .
-      - name: Setup PHP 7.1 # not included in ubuntu 18.04
-        uses: shivammathur/setup-php@9882bed06691b3a085010c1602ce43ef18f15c5b # v2
-        with:
-          php-version: '7.1'
+      # - name: Setup PHP 7.1 # not included in ubuntu 18.04
+      #   uses: shivammathur/setup-php@9882bed06691b3a085010c1602ce43ef18f15c5b # v2
+      #   with:
+      #     php-version: '7.1'
```

Comment:
remove the comment and add the linting to the list above

Reply:
https://github.com/elementor/elementor/pull/20377/commits/eccd29e265e7eb8ee0138555b0d26ac71019e025